pysat/tests/test_orbits.py

```diff
@@ -12,26 +12,48 @@
 import pysat
 
 
+def filter_data(inst, times=None):
+    """Remove data from instrument, simulating gaps in the dataset."""
+
+    if times is None:
```

Comment:
Recommend making this more robust by using the instrument index start time and adding different deltas to it.

Reply:
In order to import the instrument test date, I had to move this into the class for `TestOrbitsGappyData`.

Reply:
Updated to use `setup_class` and `teardown_class`